同步语雀文档

iShareStuff opened the pull request.

northword committed:
tweak, stage update

northword committed:
Create LICENSE

northword committed:
tweak

northword committed:
Creat CODE_OF_CONDUCT

northword committed:
Merge branch 'main' of https://github.com/zotero-chinese/wiki

northword committed:
chore: update readme [skip ci]

ImgBotApp committed:
[ImgBot] Optimize images

*Total -- 7,544.16kb -> 4,647.37kb (38.4%)

/src/assets/3c5fdf7c8321c2432a65a85f619b5172_MD5.png -- 211.60kb -> 78.13kb (63.08%)
/src/assets/01b0be7c199fb68355bffaa4a7cf5fbf_MD5.png -- 59.24kb -> 24.72kb (58.27%)
/src/assets/5250a03e83f56f9fd7107920c6804998_MD5.png -- 193.71kb -> 81.23kb (58.07%)
/src/assets/1d19f3596cbdb8d6d25d157bc6acad09_MD5.png -- 17.36kb -> 7.41kb (57.31%)
/src/assets/a92480b4cfd45829ebc23dc63236c041_MD5.png -- 44.67kb -> 22.61kb (49.39%)
/src/assets/8722848db342e9410737bc7c0c164203_MD5.png -- 30.71kb -> 15.89kb (48.25%)
/src/assets/fd27a8e887ad30a58cbba7a1f9d460fc_MD5.png -- 140.28kb -> 73.24kb (47.79%)
/src/assets/903af30f1a0d2ab93e8a6a10d156e6b4_MD5.png -- 124.16kb -> 65.24kb (47.45%)
/src/assets/9c765812f31013fc04bf2398fc64fbf2_MD5.png -- 45.76kb -> 24.29kb (46.93%)
/src/assets/6a5117508e03a5d520ca37087b071d36_MD5.png -- 148.45kb -> 79.39kb (46.52%)
/src/assets/e7605db0ea5554023c273fc47da123e1_MD5.png -- 151.67kb -> 82.51kb (45.6%)
/src/assets/09e35dd449d508c93c1a9f0431da57b8_MD5.png -- 59.58kb -> 32.79kb (44.97%)
/src/assets/b446ac93eb947050340aac9d7db76e4e_MD5.png -- 768.08kb -> 426.77kb (44.44%)
/src/assets/3ce15cf3283f04d6aa725814b60980d7_MD5.png -- 771.30kb -> 430.40kb (44.2%)
/src/assets/7012631dfe24ffda2db80b7ca842a0b8_MD5.png -- 24.58kb -> 13.76kb (44.01%)
/src/assets/c9805acb669d307e8eb8dd583b0529ee_MD5.png -- 14.71kb -> 8.37kb (43.08%)
/src/assets/3bc6c0c77292f57498336b58c2b0c4e6_MD5.png -- 433.51kb -> 252.74kb (41.7%)
/src/assets/eb56549ff9f2176c7f45d3e3d44a73f7_MD5.png -- 221.04kb -> 129.92kb (41.22%)
/src/assets/fb91af0a186aa410851bd1b112a3d12b_MD5.png -- 208.99kb -> 122.99kb (41.15%)
/src/assets/00f5ec720827811be4c00f6e3171b346_MD5.png -- 73.23kb -> 43.19kb (41.02%)
/src/assets/8edfae10d94347ddde3000af5127f21e_MD5.png -- 299.29kb -> 178.16kb (40.47%)
/src/assets/20207b568cee5c310e050b03f1ff63fd_MD5.png -- 757.46kb -> 453.59kb (40.12%)
/src/assets/ee932782423f2a644f9dface68fb895d_MD5.png -- 7.75kb -> 4.65kb (39.98%)
/src/assets/7d80c01b3f17077c51d81d0d051d08a9_MD5.png -- 50.10kb -> 30.12kb (39.88%)
/src/assets/1dbcdbf836c09ef3aca9855e85ed5c90_MD5.png -- 40.76kb -> 24.72kb (39.36%)
/src/assets/96ad3c9da3207a787e5e8f2192449711_MD5.png -- 114.63kb -> 69.78kb (39.12%)
/src/assets/c875ce553c1c51c3760cd96241de2b59_MD5.png -- 314.31kb -> 195.66kb (37.75%)
/src/assets/f29e7aedf38c994147a43e7a21e2e6fb_MD5.png -- 34.01kb -> 21.37kb (37.17%)
/src/assets/7e336d40ab0d0807f6484f54c20cbe6d_MD5.png -- 60.38kb -> 38.19kb (36.75%)
/src/assets/6976010677e1c85baaffa994ffba75a1_MD5.png -- 64.44kb -> 40.98kb (36.41%)
/src/assets/431ddd60977e41d6f5ff7b50fe321fa6_MD5.png -- 86.99kb -> 55.47kb (36.24%)
/src/assets/eeeb23f84e58bbc0d52b61956e271b72_MD5.png -- 47.57kb -> 30.37kb (36.17%)
/src/assets/009936736e7038c1a0fde4b1aa0ddcd0_MD5.png -- 112.05kb -> 73.52kb (34.38%)
/src/assets/7d78b23a4366454d9adaa3f3965b2533_MD5.png -- 92.57kb -> 62.70kb (32.27%)
/src/assets/bf72f14a344f197dc038efa443557b68_MD5.png -- 165.15kb -> 112.02kb (32.17%)
/src/assets/59b70c7e7ba02981b77b828dfe46ac81_MD5.png -- 49.21kb -> 33.67kb (31.59%)
/src/assets/7da4f4558cde17468e1b508150f0734b_MD5.png -- 18.86kb -> 12.93kb (31.44%)
/src/assets/88f260c18587f6458d9375f042bf4720_MD5.png -- 57.60kb -> 39.94kb (30.66%)
/src/assets/a245b638cfe42a492ae899724f09919d_MD5.png -- 4.90kb -> 3.57kb (27.27%)
/src/assets/4011c476a5c5af1d0e6044328c3e7ca5_MD5.png -- 129.63kb -> 96.29kb (25.72%)
/src/assets/7d7a1ca5f0b9b0e6c64a92f2172ccd07_MD5.png -- 142.67kb -> 107.32kb (24.78%)
/src/assets/2b0f6bb2614f168f4b86c4e472359e09_MD5.png -- 3.34kb -> 2.52kb (24.68%)
/src/assets/482d9614a945b8808596777b6cd0d84d_MD5.png -- 14.03kb -> 10.57kb (24.63%)
/src/assets/d81ab7808675bb480bf17fa259a0158e_MD5.png -- 10.51kb -> 7.95kb (24.36%)
/src/assets/2ef27539bcef2d06b0333859f8bff392_MD5.png -- 9.01kb -> 6.83kb (24.27%)
/src/.vuepress/public/logo.png -- 35.37kb -> 26.83kb (24.15%)
/src/assets/b6b93979415a587044988dadf36330a6_MD5.png -- 21.77kb -> 16.69kb (23.34%)
/src/assets/0def132c587ccc9c294daa8c6c1aed21_MD5.png -- 132.54kb -> 102.62kb (22.58%)
/src/assets/56405e3116e6bde1940dd4e58d937947_MD5.png -- 1.16kb -> 0.91kb (21.2%)
/src/assets/7d478dbb722fbc441b2554bcad24c090_MD5.png -- 523.17kb -> 426.03kb (18.57%)
/src/assets/f23c672cd610ade291f71bc5326bde2e_MD5.png -- 4.12kb -> 3.44kb (16.51%)
/src/assets/bb14114132259df951f23950be4c4b03_MD5.png -- 162.50kb -> 137.21kb (15.56%)
/src/assets/94e7585b0662e9b4f0019eb3c639a004_MD5.png -- 41.12kb -> 35.11kb (14.62%)
/src/assets/8725ce0123dcba87324d640b97a58a87_MD5.png -- 1.41kb -> 1.23kb (13.12%)
/src/assets/image-2.png -- 176.41kb -> 155.26kb (11.99%)
/src/assets/adccbbf1b14cbf32e7685db4574be097_MD5.png -- 0.94kb -> 0.85kb (9.76%)
/src/assets/44dc8b5f6c89fafee1bdc074d418df41_MD5.png -- 7.80kb -> 7.07kb (9.36%)
/src/assets/1682e8ea8f2e9571630f877265e3e601_MD5.png -- 1.82kb -> 1.69kb (6.93%)
/src/assets/79b28ff2fccaa647fbc2ef5ec08b77db_MD5.png -- 1.97kb -> 1.84kb (6.46%)
/src/assets/1a938ecafeaeac6dff9bb52d1968c1ba_MD5.png -- 1.03kb -> 0.98kb (4.74%)
/src/assets/image-1.png -- 1.18kb -> 1.17kb (1.08%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

northword committed:
Merge pull request #1 from zotero-chinese/imgbot

[ImgBot] Optimize images

northword committed:
添加贡献指南

northword committed:
重命名图片名

northword committed:
修改首页

northword committed:
修正首页中的错误描述

northword committed:
修复导航栏贡献者指南链接错误，贡献者指南中错误

ImgBotApp committed:
[ImgBot] Optimize images

*Total -- 629.12kb -> 415.75kb (33.91%)

/src/assets/image-build-dev-server-1.png -- 134.87kb -> 84.49kb (37.36%)
/src/assets/image-fork-repo-1.png -- 304.65kb -> 202.36kb (33.58%)
/src/assets/image-fork-repo-2-1.png -- 189.61kb -> 128.91kb (32.01%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>

northword committed:
合入 csl 文档，暂存

northword committed:
更新插件镜像页链接

northword committed:
修改CSL和首页标题

northword committed:
Add md-lint

northword committed:
Update plugins store url

northword committed:
Update Zotero Chinese domin

l0o0 committed:
doc: 去除多余的白色边框，更新图片说明

Chikit-L committed:
完善部分链接地址 (#3)

northword committed:
完善部分链接地址

Chikit-L committed:
Add files via upload (#5)

Chikit-L committed:
Update 01-在Zotero中生成参考文献.md (#4)

* Update 01-在Zotero中生成参考文献.md

完善 直接生成参考文献表 部分

* 修改图片路径，lint md

---------

Co-authored-by: Northword <northword@outlook.com>

northword committed:
新增通过VSCode-dev添加文档的指南

northword committed:
Fix Lint

imgbot[bot] committed:
[ImgBot] Optimize images (#6)

*Total -- 587.29kb -> 354.98kb (39.56%)

/src/assets/image-zotero-选择参考文献.png -- 15.53kb -> 6.58kb (57.61%)
/src/assets/image-zotero-选择引文格式及输出方式.png -- 33.24kb -> 15.88kb (52.22%)
/src/assets/image-zotero-创建参考文献表.png -- 36.53kb -> 21.88kb (40.1%)
/src/assets/image-contribution-guide-vscode1.png -- 70.59kb -> 42.45kb (39.87%)
/src/assets/image-贡献指南-进行对应的更改.png -- 285.09kb -> 171.50kb (39.84%)
/src/assets/image-贡献指南-安装插件.png -- 45.44kb -> 27.35kb (39.82%)
/src/assets/image-贡献指南-提交并推送.png -- 36.19kb -> 22.08kb (38.98%)
/src/assets/image-粘贴参考文献表.png -- 37.86kb -> 24.44kb (35.45%)
/src/assets/image-zotero官方-渲染图-connector.png -- 26.81kb -> 22.82kb (14.88%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>
Co-authored-by: ImgBotApp <ImgBotHelp@gmail.com>

northword committed:
修复提示块的渲染问题

Chikit-L committed:
完善引文格式的下载部分。 (#7)

* 完善引文格式的下载部分。

* 更新引文样式

---------

Co-authored-by: Northword <northword@outlook.com>

northword committed:
Fix typo

northword committed:
精简导航栏

northword committed:
更新VSCode粘贴图片的保存路径 [skip ci]

northword committed:
对CSCode隐藏部分无关文件 [skip ci]

iShareStuff committed:
blog-isharestuff-V0.0.1

northword commented on _resources/6633a5f8c3b54ca386551486e98caed7.png:
所有的图形资源都应放入`src/assets`

northword replied:
此外那个接近6M的图我觉得可以不放，或者找其他地方方成可编辑的在线文档。这个图打开几次就把 Netlify 的流量耗完了，我们目前每个月只有 100G 的流量。

northword commented on src/.vuepress/bulletin.js:
这是在什么地方用的？

northword commented on src/.vuepress/navbar.ts:
```diff
@@ -1,6 +1,20 @@
+ * @Date: 2023-09-15 11:09:45
+ * @LastEditors: isharestuff
+ * @LastEditTime: 2023-09-15 11:12:29
+ */
```
我觉得头部版权注释是不必要的

northword commented on src/.vuepress/navbar.ts:
```diff
@@ -1,6 +1,20 @@
+    text: "最新公告",
+    icon: "article",
+    link: "/public/readme.md",
+  },
```
vuepress-theme-hope 提供了公告板，不需要写在nav里

iShareStuff committed:
del no need js

northword commented on src/user-guide/Zotero Better Notes/0 文档简介_About.md:
成组的插件文档可以单开仓库，或是放入“插件与功能扩展”里面

northword commented on src/user-guide/生成引注_参考文献表/在 Word 中插入引文.md:
不要覆盖已有的文档，已有的文档基本已经完成了校对。

northword commented on src/user-guide/生成引注_参考文献表/在 Word 中插入引文.md:
```diff
@@ -0,0 +1,193 @@
+
+## Zotero Word 插件选项卡
+
+<img src="https://cdn.nlark.com/yuque/0/2022/png/2804696/1662207437935-9f328140-4863-4ec1-b528-51073ef2c8f3.png" width="603" id="ubd2bb440" class="ne-image">
```
这里的 cdn 是哪里提供的？可靠不？

我个人倾向于图片在仓库保存，仓库有机器人会定期压缩图片。

northword commented on src/public/README.md:
我建议 vip 网站单开仓库，不要签入 wiki 仓库，wiki 仓库目标是吸引所有人都编辑，最好不放任何和付费有关的东西。单开仓库，可以在首页或 nav 加链接过去。

northword commented on src/plugin-dev-guide/0 基础知识.md:
开发文档其实已经严重过时了，很多东西都已经不再适用于 Zotero 7 了，所以我在想把它迁移过来的必要性。

northword commented on .vscode/settings.json:
```diff
@@ -29,10 +29,18 @@
+"package.json": true,
+"package-lock.json": true,
+"CODE_OF_CONDUCT.md": true,
+"LICENSE": true
```
这里的缩进是不是被改了

northword reviewed (changes requested):
此外，目前网站依靠文件编号来实现侧边栏排序，如果这一方式不妥，我看可以改成在.vuepress/sidebar.ts里手动排序。
依靠编号自动排序需要对每一个目录都有 `README.md` 用来指定目录显示在侧边栏的名称。

northword commented on src/.vuepress/navbar.ts:
```diff
@@ -1,6 +1,20 @@
+    icon: "article",
+    link: "/public/readme.md",
+  },
   { text: "首页", icon: "home", link: "/" },
```
这一行与第12行是重复的，需要移除第12行，保留“首页”；否则会出现两个首页
![图片](https://github.com/zotero-chinese/wiki/assets/44738481/9b2cb052-8719-403a-ba58-ceedfb9d1b00)

iShareStuff committed:
modify0.0.1

iShareStuff committed:
标准化

iShareStuff committed:
init-std

northword committed:
移除`.idea`  [skip ci]
此目录是不必要的，仓库内不含有 Java 代码，针对非推荐的编辑器的个人配置不应签入仓库

northword committed:
移除无用代码 [skip ci]

northword committed:
修复JSON的格式化问题

northword committed:
添加 prettier.tabsize 配置

Chikit-L committed:
补充0.2-添加条目 (#10)

* 完善引文格式的下载部分。

* 更新引文样式

* 完成待办：从 translator_cn 仓库下载并安装 translator 的方法
补充常见术语定义。

* 修订格式
Lint，盘古空格，修复两处链接异常，删除重复图片

---------

Co-authored-by: Northword <northword@outlook.com>

imgbot[bot] committed:
[ImgBot] Optimize images (#11)

*Total -- 1,210.75kb -> 800.71kb (33.87%)

/src/assets/image-translators文件.png -- 133.26kb -> 75.50kb (43.34%)
/src/assets/image-zotero-官方引文格式.png -- 110.13kb -> 62.71kb (43.06%)
/src/assets/image-手动更新translators.png -- 90.31kb -> 53.02kb (41.3%)
/src/assets/image-zotero-获取更多引文格式.png -- 62.35kb -> 37.12kb (40.47%)
/src/assets/image-手动在github下载translators.png -- 80.05kb -> 49.39kb (38.3%)
/src/assets/image-zotero-引文格式管理.png -- 79.48kb -> 50.01kb (37.08%)
/src/assets/image-确认安装引文格式.png -- 35.91kb -> 22.91kb (36.21%)
/src/assets/image-zotero-数据储存位置.png -- 38.58kb -> 24.72kb (35.94%)
/src/assets/image-安装引文格式.png -- 49.93kb -> 33.67kb (32.56%)
/src/assets/image-浏览器中zotero设置.png -- 13.38kb -> 9.12kb (31.82%)
/src/assets/image-数据储存位置translators文件夹.png -- 11.10kb -> 7.66kb (30.98%)
/src/assets/image-connector-update-translators.png -- 48.71kb -> 34.32kb (29.54%)
/src/assets/image-github下载引文格式.png -- 457.56kb -> 340.57kb (25.57%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>
Co-authored-by: ImgBotApp <ImgBotHelp@gmail.com>

northword committed:
取消隐藏package.json
这会导致 vscode 无法正常获取到 npm 脚本

northword committed:
替换文件名为全英文，取消文件名编号

northword committed:
恢复 vscode 默认文件排序

northword committed:
测试显式指定作者

northword committed:
添加`快速开始`小节

northword committed:
减少子目录

northword committed:
将 about/ 内容移出子目录
新增贡献者列表；对未完成文档添加待办标记；完善 csl 侧边栏配置

northword committed:
修复图片路径导致的构建错误

northword committed:
修改贡献指南

northword committed:
在脚本中增加 `npm install`

northword committed:
在贡献指南添加图形化描述

northword committed:
删除安装命令避免死循环

northword committed:
添加WPS的待办

northword committed:
Merge branch 'main' into isharestuff

imgbot[bot] committed:
[ImgBot] Optimize images (#14)

*Total -- 3,084.24kb -> 2,082.53kb (32.48%)

/src/assets/image-build-提交修改-1.png -- 466.84kb -> 293.52kb (37.13%)
/src/assets/e05c956235f0b444099562d95e36a500_MD5.png -- 18.05kb -> 11.43kb (36.64%)
/src/assets/15c0d731ab0b78bf8bc194ec9e5d79a1_MD5.png -- 22.36kb -> 14.21kb (36.45%)
/src/assets/image-build-启动开发服务器-1.png -- 393.38kb -> 251.86kb (35.98%)
/src/assets/image-build-推送修改-1.png -- 400.88kb -> 257.33kb (35.81%)
/src/assets/image-build-创建新分支-1.png -- 395.46kb -> 254.66kb (35.61%)
/src/assets/image-build-安装依赖-1.png -- 306.31kb -> 198.28kb (35.27%)
/src/assets/f38b68d04c167de25cd6e1f59b3dea59_MD5.png -- 19.94kb -> 13.24kb (33.6%)
/src/assets/56bb224283b259e7ad53bba8c4074d12_MD5.png -- 16.60kb -> 11.10kb (33.15%)
/src/assets/741b873973ac5df93faf93cd1a195db2_MD5.png -- 21.24kb -> 14.35kb (32.46%)
/src/assets/image-build-更新main分支-1.png -- 187.13kb -> 130.16kb (30.45%)
/src/assets/8c581efa5ed9f3ac9d5771b89204bd6b_MD5.png -- 181.95kb -> 129.00kb (29.1%)
/src/assets/image-build-pr预览-1.png -- 96.60kb -> 68.53kb (29.06%)
/src/assets/28f02608aa879ed189ecfecfcab57d16_MD5.png -- 236.91kb -> 169.03kb (28.65%)
/src/assets/5db26b70fa394ac4b20c8f9702899bc6_MD5.png -- 20.06kb -> 14.47kb (27.85%)
/src/assets/d5cc90422e3df7d03c9a75b458c6cde1_MD5.png -- 48.20kb -> 35.90kb (25.53%)
/src/assets/47516f8b73477847a13a2c861e80ed10_MD5.png -- 7.22kb -> 5.50kb (23.88%)
/src/assets/9383416a28f689e820457d0baf6d4158_MD5.png -- 7.67kb -> 5.95kb (22.4%)
/src/assets/1a7ad705eb230c7ba84bd4f40fc9b75d_MD5.png -- 13.62kb -> 10.70kb (21.46%)
/src/assets/7f7e2dca1e509cf53dffb18431304d0c_MD5.png -- 10.91kb -> 8.69kb (20.4%)
/src/assets/a81c5f82bdf314dc4b746e5265fe0773_MD5.png -- 17.65kb -> 14.21kb (19.53%)
/src/assets/033a4684b6867c7e4dc87acd6423c99a_MD5.png -- 7.80kb -> 6.33kb (18.82%)
/src/assets/e32b58ee1c85587c51497c81de98d39a_MD5.png -- 8.28kb -> 6.73kb (18.69%)
/src/assets/60cbda53f8de56de50bdb15d9dfc8b78_MD5.png -- 10.53kb -> 8.64kb (17.95%)
/src/assets/80cebc82ab653f229fecd246e695ed10_MD5.png -- 14.71kb -> 12.21kb (16.98%)
/src/assets/474334b603ae0913ad81a0ddd1dcbe91_MD5.png -- 10.49kb -> 8.78kb (16.34%)
/src/assets/cd4a48155186f0307118c55deb327ed4_MD5.png -- 26.66kb -> 22.66kb (14.99%)
/src/assets/46510f422020d39fa7a495030129abe3_MD5.png -- 21.53kb -> 18.45kb (14.33%)
/src/assets/d755599b6302d243d9938064d33f45a4_MD5.png -- 10.76kb -> 9.35kb (13.15%)
/src/assets/e99c949fbf9a48dab6305a9244ea7a8a_MD5.png -- 27.04kb -> 23.49kb (13.14%)
/src/assets/fec738499e295fe7967677534303846a_MD5.png -- 15.35kb -> 13.55kb (11.69%)
/src/assets/9d07a67981c445f862fa46bddbfcd0db_MD5.png -- 6.68kb -> 5.97kb (10.53%)
/src/assets/1a2a19b3c0e17fbb4805d0b53ff1b1b0_MD5.png -- 19.46kb -> 18.65kb (4.19%)
/src/assets/f12861c197d522f4d27795857b2eb167_MD5.png -- 15.94kb -> 15.62kb (2.02%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>
Co-authored-by: ImgBotApp <ImgBotHelp@gmail.com>

northword committed:
暂存：移动文件

northword committed:
暂存：修复部分格式

northword committed:
暂存：修改部分格式

northword committed:
修改侧边栏，增加webdav和zoterobib

Chikit-L committed:
添加 图片上有黑色遮挡的解决方法 (#15)

* 添加 图片上有黑色遮挡的解决方法
修改 手动更新translators中的部分错误
修改 手动更新translators中的部分错误 文件名

* 修改格式

* 更新格式

---------

Co-authored-by: Northword <northword@outlook.com>

northword committed:
增加百度统计，更新依赖 (#13) 

* 添加百度统计

* Update deps

* 删除 npm 锁定文件

* 移除环境变量，环境变量需要在netlify添加

* 添加百度统计Key

* 非生产环境不引入百度统计

northword committed:
修复百度统计不生效的问题

northword committed:
不再隐藏文件

northword committed:
对PR测试能否成功构建 (#20)

* 对PR测试能否成功构建

* 测试取消npm缓存以修复ci错误

* 修复取消使用缓存导致的安装脚本错误

northword committed:
重命名build action名称

northword committed:
Fix lint (#19)

* 修复部分lint错误

* 添加自动lint检测action

* 修正lint范围

* 修复部分死链

northword committed:
Fix build error

Chikit-L committed:
部分完成数据与文件的同步 (#18)

* 添加 图片上有黑色遮挡的解决方法
修改 手动更新translators中的部分错误
修改 手动更新translators中的部分错误 文件名

* 修改格式

* 更新格式

* 完成部分 数据与文件的同步

* 修改部分格式

* 修复图片名大小写错误导致的构建失败

* 修复图片名大小写错误

---------

Co-authored-by: Northword <northword@outlook.com>

l0o0 committed:
添加WPS插件安装及使用教程 (#16)

* 添加WPS插件安装及使用教程

* 更新 WPS 使用 Zotero 参考文献文档

* 修改部分格式

* 修改部分格式

* 去除首页中的句号

---------

Co-authored-by: Northword <northword@outlook.com>

imgbot[bot] committed:
[ImgBot] Optimize images (#17)

*Total -- 184.78kb -> 113.25kb (38.71%)

/src/assets/image-zotero-黑色遮挡.png -- 92.05kb -> 46.97kb (48.97%)
/src/assets/image-zotero-我同意承担风险.png -- 31.55kb -> 22.07kb (30.04%)
/src/assets/image-zotero-编辑器.png -- 33.75kb -> 23.85kb (29.33%)
/src/assets/image-zotero-layers.acceleration.disabled.png -- 27.44kb -> 20.36kb (25.79%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>
Co-authored-by: ImgBotApp <ImgBotHelp@gmail.com>

northword committed:
Merge branch 'main' into isharestuff

imgbot[bot] committed:
[ImgBot] Optimize images (#23)

*Total -- 6,693.09kb -> 496.73kb (92.58%)

/src/assets/image-wps-success.png -- 1,352.75kb -> 9.38kb (99.31%)
/src/assets/image-wps-install.png -- 327.14kb -> 3.27kb (99%)
/src/assets/wps.png -- 1,322.46kb -> 23.61kb (98.22%)
/src/assets/image-wps-plugin-ui-buttons.png -- 317.95kb -> 8.26kb (97.4%)
/src/assets/image-run-python-version.png -- 433.18kb -> 13.19kb (96.95%)
/src/assets/image-wps-plugin-ui.png -- 300.75kb -> 9.24kb (96.93%)
/src/assets/image-run-cmd.png -- 267.78kb -> 8.62kb (96.78%)
/src/assets/image-miniconda-install.png -- 568.62kb -> 19.23kb (96.62%)
/src/assets/image-wps-vba-plugin-install-path.png -- 366.49kb -> 13.13kb (96.42%)
/src/assets/wps-pluginfile-list.png -- 486.33kb -> 24.37kb (94.99%)
/src/assets/image-python-install.png -- 707.09kb -> 175.77kb (75.14%)
/src/assets/image-坚果云-生成密码.png -- 5.92kb -> 2.35kb (60.31%)
/src/assets/image-坚果云-密码.png -- 6.46kb -> 2.59kb (59.94%)
/src/assets/image-wps-warn2.png -- 21.98kb -> 12.10kb (44.94%)
/src/assets/image-wps-warn1.png -- 18.50kb -> 10.37kb (43.93%)
/src/assets/image-zotero-帐号登陆.png -- 22.12kb -> 17.24kb (22.08%)
/src/assets/image-坚果云-账户信息.png -- 6.69kb -> 5.47kb (18.26%)
/src/assets/image-zotero-服务器验证成功.png -- 6.44kb -> 5.45kb (15.49%)
/src/assets/image-坚果云-添加应用.png -- 70.47kb -> 59.70kb (15.29%)
/src/assets/image-zotero-登陆webdav账户.png -- 45.87kb -> 39.22kb (14.49%)
/src/assets/image-zotero-webdav网址设置.png -- 38.11kb -> 34.18kb (10.33%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>
Co-authored-by: ImgBotApp <ImgBotHelp@gmail.com>

Chikit-L committed:
完成ios同步部分内容 (#22)

* 添加 图片上有黑色遮挡的解决方法
修改 手动更新translators中的部分错误
修改 手动更新translators中的部分错误 文件名

* 修改格式

* 更新格式

* 完成部分 数据与文件的同步

* 修改部分格式

* 修复图片名大小写错误导致的构建失败

* 修复图片名大小写错误

* 完成ios同步部分内容

* 修改部分格式

---------

Co-authored-by: Northword <northword@outlook.com>

l0o0 committed:
修复了部分文章中连续图片不能居中的问题，更新了zoo同步教程 (#25)

* fix: 修复连续图片不居中问题

* add: 添加Android 客户端 Zoo 的同步设置

* 修改大小写和空格

* 修改无序列表格式

* Fix Lint

---------

Co-authored-by: Northword <44738481+northword@users.noreply.github.com>

imgbot[bot] committed:
[ImgBot] Optimize images (#24)

*Total -- 132.25kb -> 104.62kb (20.9%)

/src/assets/image-iOS登陆.jpg -- 104.75kb -> 77.81kb (25.71%)
/src/assets/image-iOS首页.png -- 27.51kb -> 26.80kb (2.56%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>
Co-authored-by: ImgBotApp <ImgBotHelp@gmail.com>

imgbot[bot] committed:
[ImgBot] Optimize images (#26)

*Total -- 1,764.57kb -> 1,661.36kb (5.85%)

/src/assets/image-zoo-webdav设置.png -- 487.85kb -> 429.91kb (11.88%)
/src/assets/image-zoo加载数据.png -- 97.54kb -> 89.15kb (8.6%)
/src/assets/image-zoo添加zotero密钥.png -- 547.22kb -> 521.48kb (4.7%)
/src/assets/image-zoo-download-pdf.png -- 631.96kb -> 620.81kb (1.76%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>
Co-authored-by: ImgBotApp <ImgBotHelp@gmail.com>

northword committed:
在贡献者指南中添加新建分支相关描述 (#29)

imgbot[bot] committed:
[ImgBot] Optimize images (#30)

*Total -- 162.58kb -> 97.54kb (40%)

/src/assets/image-contribution-new-branch-name.png -- 22.02kb -> 11.84kb (46.24%)
/src/assets/image-contribute-vscode-new-branch.png -- 85.48kb -> 51.79kb (39.41%)
/src/assets/image-contribution-switch-to-new-branch.png -- 55.08kb -> 33.91kb (38.44%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>
Co-authored-by: ImgBotApp <ImgBotHelp@gmail.com>

northword committed:
添加Giscus评论 (#31)

* 添加Giscus评论系统

* 修改讨论区链接

northword committed:
chore: 更新VS Code配置 [skip ci]

northword committed:
添加备份小节 (#28)



Co-authored-by: Chikit-L <529158751@qq.com>

northword committed:
Merge branch 'main' into isharestuff

northword committed:
移除已有内容

northword committed:
暂时移除开发文档

northword committed:
Markdown Lint Fix

northword committed:
添加其他系统的zotero profile directory

northword committed:
移动行为准则至网站

northword committed:
修改开源协议为CC BY-SA 4.0

northword committed:
添加关于word引用使用的一些文章 (#32)

* 添加关于word引用使用的一些文章

* 更新元信息

northword committed:
Refactor GitHub action workflows (#47)

imgbot[bot] committed:
[ImgBot] Optimize images (#33)

*Total -- 7,186.93kb -> 5,415.29kb (24.65%)

/src/assets/image-word-域代码.png -- 243.18kb -> 98.63kb (59.44%)
/src/assets/image-endnote-link-citation-to-bib.png -- 46.58kb -> 28.39kb (39.04%)
/src/assets/f30e9f8627698b66e84c9bdc9a595663_MD5.png -- 1,050.61kb -> 687.08kb (34.6%)
/src/assets/1e7deb7a378eadbeda2d05f2ed6408ca_MD5.png -- 601.28kb -> 395.75kb (34.18%)
/src/assets/e2f9f84e616a8113ce86de71f95e6764_MD5.png -- 865.56kb -> 596.13kb (31.13%)
/src/assets/image-word-确认保存修改.png -- 44.21kb -> 30.89kb (30.13%)
/src/assets/fb54e92ec7ed8c0a631028f76695a78a_MD5.png -- 734.44kb -> 514.89kb (29.89%)
/src/assets/a2f3237d76883f049b272f29abdf0c51_MD5.png -- 201.75kb -> 160.07kb (20.66%)
/src/assets/50a0fa1a34f38c62dcbd4cc31b579087_MD5.png -- 88.77kb -> 71.56kb (19.39%)
/src/assets/fcbe2e6cdb021edcad348da71e947a2a_MD5.png -- 145.61kb -> 119.17kb (18.16%)
/src/assets/4de80b5cd85863024d63689cdf806721_MD5.png -- 67.26kb -> 55.21kb (17.92%)
/src/assets/5ff7f644bdf93f26d18e48f2f0ee3c43_MD5.png -- 127.23kb -> 104.48kb (17.88%)
/src/assets/image-tools-addon.png -- 269.83kb -> 222.18kb (17.66%)
/src/assets/8addbd81b3d85f5501f1a58020ab3191_MD5.png -- 107.40kb -> 88.44kb (17.66%)
/src/assets/6dd98ac109ac059509e72720c885e733_MD5.png -- 77.62kb -> 64.24kb (17.25%)
/src/assets/2a4390faa87ca714c9264b8f3bae4b64_MD5.png -- 131.77kb -> 110.72kb (15.97%)
/src/assets/81ab37e57b20cd4223c7e3b4c42f0232_MD5.png -- 142.80kb -> 120.22kb (15.81%)
/src/assets/67c928f040091835a073775ea418ff4c_MD5.png -- 70.25kb -> 59.21kb (15.72%)
/src/assets/df5d7f252fb711f5362abd06c293f8fd_MD5.png -- 183.70kb -> 155.61kb (15.29%)
/src/assets/image-zotero-addon-manager.png -- 83.76kb -> 71.01kb (15.22%)
/src/assets/aab13e38e1dbeb28f675129f45f38af8_MD5.png -- 172.17kb -> 146.42kb (14.96%)
/src/assets/4a3c7755cb85eb19bc614090a3f08676_MD5.png -- 105.67kb -> 90.25kb (14.6%)
/src/assets/image-settings-reinstalll-word-plugin.png -- 57.55kb -> 49.25kb (14.44%)
/src/assets/6b8137561c99a0795bdcb48f57fab85e_MD5.png -- 200.40kb -> 172.10kb (14.12%)
/src/assets/a24bbdd6425f0d2aea3a7f34a0989386_MD5.png -- 173.48kb -> 150.04kb (13.51%)
/src/assets/d8772b1c75f067c9322b337c5b6fe717_MD5.png -- 158.00kb -> 137.26kb (13.13%)
/src/assets/image-wps-没有显示加载项.png -- 312.12kb -> 272.58kb (12.67%)
/src/assets/d566856af09a69bd03db196b1e3744cf_MD5.png -- 89.45kb -> 78.24kb (12.53%)
/src/assets/7b352103fed9e315ab8c3543f0ceb4f1_MD5.png -- 108.33kb -> 95.00kb (12.3%)
/src/assets/b8282b0cdce33505057400c176ea2cc5_MD5.png -- 106.51kb -> 93.49kb (12.22%)
/src/assets/15e71e5849e591f2252b7e706e8d6707_MD5.png -- 248.52kb -> 218.63kb (12.03%)
/src/assets/8a33ca40095a62ad5b3c6857dc6add55_MD5.png -- 69.07kb -> 63.04kb (8.73%)
/src/assets/a01d83b2dbe0f1d2c356c891336416ec_MD5.png -- 102.02kb -> 95.12kb (6.76%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>
Co-authored-by: ImgBotApp <ImgBotHelp@gmail.com>

northword committed:
添加 `zbib.org` 用法 (#49)

* 添加 `zbib.org` 用法

* 补充描述

* 调整侧边栏顺序

* 添加推荐插件

* 修改图片alt文本

northword committed:
部署前签出仓库

northword committed:
Merge branch 'main' of https://github.com/zotero-chinese/wiki

northword committed:
修复死链，添加代码高亮行 (#50)

* 修复死链二处

* 添加代码高亮行

Chikit-L committed:
emoji变成黑白的解决方法 (#52)

* emoji变成黑白

* Fix lint

---------

Co-authored-by: Northword <northword@outlook.com>

yeah-zihan committed:
Fix typo in src/README.md (#53)

l0o0 committed:
修改 iOS 示例图尺寸 (#54)

修改iOS 示例图尺寸为宽度450

Chikit-L committed:
获取更多样式”打不开怎么办 (#55)

* “获取更多样式”打不开怎么办

* Fix lint

---------

Co-authored-by: Northword <northword@outlook.com>

imgbot[bot] committed:
[ImgBot] Optimize images (#51)

*Total -- 393.15kb -> 270.12kb (31.29%)

/src/assets/image-bib-选择样式.png -- 162.54kb -> 103.90kb (36.08%)
/src/assets/image-bib-复制结果.png -- 144.75kb -> 99.04kb (31.58%)
/src/assets/image-bib-首页.png -- 85.86kb -> 67.19kb (21.75%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>
Co-authored-by: ImgBotApp <ImgBotHelp@gmail.com>

Chikit-L committed:
完成插件备份部分 (#57)

* 完成插件备份部分

* 修改格式

* 修改格式，增加tara简介和下载链接

---------

Co-authored-by: Northword <northword@outlook.com>

northword committed:
修改tara备份locate的描述 (#59)

windingwind committed:
Add plugin install (#61)

* Add plugin install

https://github.com/zotero-chinese/wiki/issues/45

* Fix lint

* style: pangu space

---------

Co-authored-by: Northword <northword@outlook.com>

Chikit-L committed:
参考文献中论文题目部分单词如何实现斜体及上标、下标效果 (#60)

* 参考文献中论文题目部分单词如何实现斜体及上标、下标效果

* Fix lint，添加链接，替换图片为文本

* 添加删除线

---------

Co-authored-by: Northword <northword@outlook.com>

imgbot[bot] committed:
[ImgBot] Optimize images (#62)

*Total -- 867.72kb -> 564.58kb (34.94%)

/src/assets/image-zotero-引文格式安装方式.png -- 7.06kb -> 3.03kb (57.17%)
/src/assets/image-Tara-导出的备份.png -- 9.43kb -> 4.39kb (53.4%)
/src/assets/image-通过在题目中添加代码实现下标效果.png -- 12.89kb -> 6.52kb (49.46%)
/src/assets/image-Tara-导出备份成功.png -- 23.41kb -> 12.38kb (47.11%)
/src/assets/image-Tara-备份恢复完成.png -- 13.98kb -> 7.42kb (46.94%)
/src/assets/image-zotero-禁用插件.png -- 214.26kb -> 122.14kb (42.99%)
/src/assets/image-Tara-备份文件.png -- 3.86kb -> 2.34kb (39.28%)
/src/assets/image-zotero-引文格式下载.png -- 79.49kb -> 49.37kb (37.89%)
/src/assets/image-获取更多样式没有反应.png -- 74.47kb -> 47.57kb (36.13%)
/src/assets/image-zotero-安装本地引文格式.png -- 73.80kb -> 47.20kb (36.04%)
/src/assets/image-Tara-选择要恢复的备份.png -- 10.59kb -> 7.09kb (33.07%)
/src/assets/image-Tara-导出备份.png -- 16.19kb -> 10.93kb (32.49%)
/src/assets/image-Tara-恢复备份.png -- 15.35kb -> 10.54kb (31.33%)
/src/assets/image-zotero-引文格式安装完成.png -- 2.18kb -> 1.50kb (30.94%)
/src/assets/image-zotero-修改font.name-list.emoji.png -- 60.92kb -> 42.42kb (30.37%)
/src/assets/image-Refresh.png -- 9.24kb -> 6.50kb (29.71%)
/src/assets/image-zotero-plugin-install.png -- 170.75kb -> 122.14kb (28.47%)
/src/assets/image-Tara-创建备份.png -- 11.33kb -> 8.39kb (25.89%)
/src/assets/image-引文格式下载.png -- 4.14kb -> 3.15kb (23.88%)
/src/assets/image-emoji由黑白变成彩色.jpg -- 2.12kb -> 1.68kb (20.72%)
/src/assets/image-zotero-退出账户.png -- 48.32kb -> 44.14kb (8.66%)
/src/assets/image-emoji变成黑白.jpg -- 3.94kb -> 3.74kb (4.89%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>
Co-authored-by: ImgBotApp <ImgBotHelp@gmail.com>

northword committed:
增加Feed小节 (#64)

Co-authored-by: Comte0825 <895556682@qq.com>

l0o0 committed:
添加从其他来源导入Zotero, close #43 (#65)

* 添加从其他来源导入zotero, #43

* Fix lint

* 移动到单独文件

* 修改描述

---------

Co-authored-by: Comte0825 <895556682@qq.com>
Co-authored-by: Northword <northword@outlook.com>

imgbot[bot] committed:
[ImgBot] Optimize images (#66)

*Total -- 20,113.04kb -> 14,169.69kb (29.55%)

/src/assets/image-feed-24.png -- 444.50kb -> 196.31kb (55.84%)
/src/assets/image-feed-3.png -- 19.80kb -> 9.73kb (50.85%)
/src/assets/image-feed-12.png -- 677.24kb -> 365.52kb (46.03%)
/src/assets/image-feed-6.png -- 122.67kb -> 68.54kb (44.13%)
/src/assets/image-feed-26.png -- 386.33kb -> 226.77kb (41.3%)
/src/assets/endnote-zotero-import-option2.png -- 94.01kb -> 59.61kb (36.6%)
/src/assets/word-zotero-succ.png -- 541.14kb -> 343.96kb (36.44%)
/src/assets/image-feed-22.png -- 226.40kb -> 147.77kb (34.73%)
/src/assets/image-feed-5.png -- 728.13kb -> 477.16kb (34.47%)
/src/assets/endnote-zotero-caution.png -- 248.93kb -> 163.36kb (34.37%)
/src/assets/endnote-zotero-import-option.png -- 101.93kb -> 66.96kb (34.31%)
/src/assets/endnote-export.png -- 1,499.87kb -> 991.03kb (33.93%)
/src/assets/endnote-menu.png -- 1,534.32kb -> 1,017.07kb (33.71%)
/src/assets/image-feed-18.png -- 553.35kb -> 368.78kb (33.36%)
/src/assets/word-export-reference.png -- 776.63kb -> 529.20kb (31.86%)
/src/assets/word-export-reference-copy.png -- 819.19kb -> 558.76kb (31.79%)
/src/assets/word-finder-path.png -- 177.45kb -> 121.27kb (31.66%)
/src/assets/endnote-save.png -- 278.72kb -> 191.07kb (31.45%)
/src/assets/image-feed-20.png -- 628.44kb -> 436.09kb (30.61%)
/src/assets/image-feed-7.png -- 124.95kb -> 87.35kb (30.1%)
/src/assets/word-zotero-import.png -- 283.07kb -> 198.27kb (29.96%)
/src/assets/image-feed-17.png -- 770.04kb -> 552.67kb (28.23%)
/src/assets/word-export-reference-format.png -- 862.62kb -> 619.50kb (28.18%)
/src/assets/endnote-zotero-import.png -- 216.26kb -> 155.58kb (28.06%)
/src/assets/image-feed-10.png -- 155.18kb -> 112.23kb (27.68%)
/src/assets/image-feed-2.png -- 194.95kb -> 141.61kb (27.36%)
/src/assets/word-export-folder.png -- 366.15kb -> 268.74kb (26.6%)
/src/assets/image-feed-15.png -- 2,220.35kb -> 1,629.68kb (26.6%)
/src/assets/image-feed-21.png -- 420.78kb -> 310.31kb (26.25%)
/src/assets/word-copy-bib.png -- 411.71kb -> 303.73kb (26.23%)
/src/assets/endnote-zotero-succ.png -- 382.11kb -> 285.48kb (25.29%)
/src/assets/image-feed-16.png -- 768.08kb -> 589.07kb (23.31%)
/src/assets/image-feed-11.png -- 300.94kb -> 238.00kb (20.91%)
/src/assets/endnote-toxml.png -- 47.68kb -> 37.86kb (20.61%)
/src/assets/endnote-xml.png -- 551.71kb -> 438.66kb (20.49%)
/src/assets/image-feed-14.png -- 740.65kb -> 596.86kb (19.41%)
/src/assets/image-feed-25.png -- 626.63kb -> 521.91kb (16.71%)
/src/assets/image-feed-9.png -- 21.64kb -> 18.34kb (15.26%)
/src/assets/image-feed-13.png -- 209.99kb -> 188.99kb (10%)
/src/assets/image-feed-23.png -- 253.66kb -> 234.42kb (7.58%)
/src/assets/endnote-xml-open.png -- 324.87kb -> 301.52kb (7.19%)

Signed-off-by: ImgBotApp <ImgBotHelp@gmail.com>
Co-authored-by: ImgBotApp <ImgBotHelp@gmail.com>

northword committed:
更新百度统计

northword committed:
修复百度统计未生效的问题

northword committed:
修改评论区内容存储仓库 (#70)

将 discus 的存储仓库从 .github 迁移到 wiki，有如下考虑：
- .github 仓库访问量/star/watch 都过少
- 读者倾向于在 wiki 创建 issue 来咨询问题等，这类 issue 转为 discussion 时仅能到 wiki 仓库
- 已经 star/watch wiki 仓库的用户在新 discussion 被创建时，可能会收到提醒，有利于提高互动度。

northword committed:
提交dist到`zotero-chinese.github.io` (#95)

* 提交dist到`zotero-chinese.github.io`

* 更新TOKEN [skip netlify]

* 修复错误

* 压缩构建以缩短上传时间，添加name

* 修复构建失败

* 启用单次提交

* 仅在main变更时部署

northword committed:
完善一些文档语法风格 (#94)

* 完善一些文档语法风格

* 补充修改

Chikit-L committed:
Zotero在更新文档时出错 (#96)

* Zotero在更新文档时出错

* 移动到 `word-addon.md`, 添加几个小节的链接

* 移动添加上下标小节

---------

Co-authored-by: Northword <northword@outlook.com>

northword committed:
重命名COC为小写，修改README格式 (#97)

* 重命名COC为小写，修改README格式

* 添加badge

northword committed:
调整页面样式：加粗三四级标题 (#101)

northword committed:
修复图片大小错误 (#99)

* 修复图片大小错误

* 恢复启用GFM语法

northword committed:
更新依赖，README增加仓库大小 (#102)

* 更新依赖

* 添加仓库大小徽标

northword committed:
对PR压缩图片 (#104)

northword committed:
切换包管理器为pnpm，启用CI缓存 (#103)

* 缓存依赖项

* 修改为pnpm

* 指定pnpm版本

* 在package-json中指定pnpm

* 合并step

* Revert "合并step"

This reverts commit ba54b9fe4bef3358386efb22bffb2377f7220017.

wakewon committed:
完善添加条目相关文档 (#100)

* 完善添加条目文档，完善更新translators文档，完善安装插件文档，添加进阶教程类别

* 修订标题，完善视觉效果及文档细节，整理文档

* Fix lint

* Fix lint [skip netlify]
[skip netlify]

* 调整部分内容的格式和顺序

* fix typo

* 删除未使用的图片

---------

Co-authored-by: Northword <northword@outlook.com>

northword committed:
维护：压缩PR中的所有图片

Chikit-L committed:
完善插件的下载安装和更新部分 (#106)

* 完善插件的安装和更新部分

* 裁剪图片去掉黑边，修改描述

* 裁剪图片

---------

Co-authored-by: Northword <44738481+northword@users.noreply.github.com>

northword committed:
维护：增加Prettier和Git Hooks (#107)

* 添加prettier和git hook

* Fix lint

* Disable KaTeX

* Fix typo

* Add comment to markdownlint config
[skip ci]
[skip netlify]

* 从lint-stage移除markdownlint

* 修复大小写错误

* reorder settings

* 仅在本仓库触发部署，fork仓库跳过

Comte0825 committed:
添加”群组“小节 (#63)

* Update group.md

In progress

* Rewrite some sections

* stage

* 转移加入群组和管理群组部分

* tweak

---------

Co-authored-by: Northword <northword@outlook.com>

northword committed:
Merge branch 'main' into isharestuff

northword committed:
lint & 移除过时的文档
